modules/get-started/pages/install-beta.adoc

```diff
@@ -0,0 +1,154 @@
+      # Use the internal addresses to connect to the Redpanda brokers'
+      # from inside the same Docker network.
+      # Use the external addresses to connect to the Redpanda brokers'
+      # from outside the Docker network.
+      - --advertise-kafka-addr internal://redpanda-0:9092,external://localhost:19092
+      - --pandaproxy-addr internal://0.0.0.0:8082,external://0.0.0.0:18082
+      # Address the broker advertises to clients that connect to the HTTP Proxy.
+      - --advertise-pandaproxy-addr internal://redpanda-0:8082,external://localhost:18082
+      - --schema-registry-addr internal://0.0.0.0:8081,external://0.0.0.0:18081
+      # Redpanda brokers use the RPC API to communicate with eachother internally.
+      - --rpc-addr redpanda-0:33145
+      - --advertise-rpc-addr redpanda-0:33145
+      # Tells Seastar (the framework Redpanda uses under the hood) to use 1 core on the system.
+      - --smp 1
+      # The amount of memory to make available to Redpanda.
+      - --memory 1G
+      # Mode dev-container uses well-known configuration properties for development in containers.
+      - --mode dev-container
+      # enable logs for debugging.
+      - --default-log-level=debug
+	image: docker.redpanda.com/redpandadata/redpanda-unstable:v23.3.1-rc1
+    container_name: redpanda-0
+    volumes:
+      - redpanda-0:/var/lib/redpanda/data
+    networks:
+      - redpanda_network
+    ports:
+      - 18081:18081
+      - 18082:18082
+      - 19092:19092
+      - 19644:9644
+  console:
+    container_name: redpanda-console
+    image: docker.redpanda.com/redpandadata/console:v2.3.7
+    networks:
+      - redpanda_network
+    entrypoint: /bin/sh
+    command: -c 'echo "$$CONSOLE_CONFIG_FILE" > /tmp/config.yml; /app/console'
+    environment:
+      CONFIG_FILEPATH: /tmp/config.yml
+      CONSOLE_CONFIG_FILE: |
+        kafka:
+          brokers: ["redpanda-0:9092"]
+          schemaRegistry:
+            enabled: true
+            urls: ["http://redpanda-0:8081"]
+        redpanda:
+          adminApi:
+            enabled: true
+            urls: ["http://redpanda-0:9644"]
+    ports:
+      - 8080:8080
+    depends_on:
+      - redpanda-0
+```
+====
+
+--
+Linux::
++
+--
+
+. Add the `redpanda-unstable` repository to your package manager:
++
+```
+curl -1sLf \
+  'https://dl.redpanda.com/E4xN1tVe3Xy60GTx/redpanda-unstable/setup.deb.sh' \
+```
+. Install Redpanda:
++
+```
+sudo apt-get install redpanda
+```
+
+* See https://hub.docker.com/r/redpandadata/redpanda-unstable/tags[dockerhub^] for the list of RC builds.
+* See the https://github.com/redpanda-data/redpanda#release-candidate-builds[readme for RC builds^] for distribution-specific information. 
+
+--
+Kubernetes::
++
+--
+
+Pull the 23.3 RC build from https://hub.docker.com/r/redpandadata/redpanda-unstable/tags[dockerhub^]. For example:
+
+```
+docker pull redpandadata/redpanda-unstable:v23.3.1-rc1
+```
+
+For example, with Helm:
```

Comment:
I don't this is how we want to describe this step, which is "install Redpanda with helm using the Redpanda-unstable docker image repo "

I don't think we need the previous step at all , right @alejandroEsc ?

Reply:
I updated this to Install Redpanda with Helm from the RC build in [dockerhub](https://hub.docker.com/r/redpandadata/redpanda-unstable/tags):

helm repo add redpanda https://charts.redpanda.com
helm install redpanda redpanda/redpanda \
  --namespace <namespace> \
  --create-namespace --set image.repository=docker.redpanda.com/redpandadata/redpanda-unstable --set image.tag=v23.3.1-rc1